src/server.rs

```diff
@@ -37,6 +37,7 @@ pub async fn run(
         .manage(callback)
         .launch()
         .await
+        .map(drop)
```

Comment:
What does this do?

Reply:
Rocket changed the return value from `()` to something else. `drop` is a function that takes any argument and returns `()`.

Reply:
Now I changed the syntax to something more readable.